src/popup/components/Modals/ConfirmTransactionSign.vue

```diff
@@ -194,6 +197,12 @@ export default defineComponent({
     const getTxSymbol = useGetter('getTxSymbol');
     const getTxAmountTotal = useGetter('getTxAmountTotal');
 
+    const tokensLoaded = computed(
+      () => (isDex.value || isAllowance.value)
```

Comment:
Right now this returns `number | false` but should return boolean I guess.

Reply:
You're right, I'll change it

Reply:
Could change it to `isTokensLoaded`